flyteadmin/auth/authzserver/metadata_provider.go

```diff
@@ -71,10 +80,15 @@ func (s OAuth2MetadataProvider) GetOAuth2Metadata(ctx context.Context, r *servic
 			}
 			httpClient.Transport = transport
 		}
-
-		response, err := httpClient.Get(externalMetadataURL.String())
+		logger.Info(ctx, "retryAttempts: %v retryDuration: %v", s.cfg.AppAuth.ExternalAuthServer.RetryAttempts, s.cfg.AppAuth.ExternalAuthServer.RetryDelay)
```

Comment:
did you want to keep this?

Reply:
For now yes, just wanting to bake it in staging for a while to try to capture a retry :)